src/test/java/htsjdk/variant/variantcontext/writer/VCFWriterUnitTest.java

```diff
@@ -77,28 +80,21 @@
     @BeforeClass
     private void createTemporaryDirectory() {
         tempDir = TestUtil.getTempDirectory("VCFWriter", "StaleIndex");
+        tempDir.deleteOnExit();
```

Comment:
I'm not sure this pattern works.  The older `deleteTemporaryDirectory` method that was called previously seems more likely to work.  I would just mark all of the temporary files as `deleteOnExit()`